catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -358,15 +360,14 @@ protected Object executeWithSubject() throws Exception {
     private void exportMetacard(File dumpLocation, Metacard metacard)
             throws IOException, CatalogTransformerException {
 
-        if (DEFAULT_TRANSFORMER_ID.matches(transformerId)) {
+        if (SERIALIZED_OBJECT_ID.matches(transformerId)) {
```

Comment:
Seems weird to have the default for ingest be XML but the default for dump is still serializable.  We should keep them consistent.